docs/index.asciidoc

```diff
@@ -0,0 +1,221 @@
+[[plugins-filters-dissect]]
```

Comment:
Why `index.asciidoc` instead of naming the file according to the plugin name? I'm trying to imagine how the asciidoc files will be copied over to one place and referenced in the TOC. What is the final structure that the doc build will see? What will we see under `/docs/plugins/filters`? Surely not `logstash-filter-dissect/docs/index.asciidoc`? Or will the copy operation rename the file or something?